src/screens/feed/components/FeedPost/FeedPost.tsx

```diff
@@ -18,11 +19,37 @@ type FeedPostProps = {
 const AnimatedBox = Animated.createAnimatedComponent(Box)
 
 export const FeedPost = ({ post }: FeedPostProps) => {
+  const [showBorder, setShowBorder] = useState(false)
   const animProgress = useSharedValue(post.recentlyAdded ? 0 : 11)
+  const navigation = useNavigation<any>()
+
   useEffect(() => {
     animProgress.value = withTiming(11, { duration: 800, easing: Easing.exp })
   }, [animProgress])
 
+  useFocusEffect(
+    React.useCallback(() => {
+      const { routes } = navigation.getState()
+      const getIdParam = routes.slice().pop()?.params?.postId
+      const getPrevScreen = routes.slice().pop()?.params?.prevScreen
+
+      const isFromNotifications =
+        getPrevScreen === 'NOTIFICATIONS' && getIdParam === Number(post.id)
+
+      if (isFromNotifications) {
+        setShowBorder(true)
+
+        setTimeout(() => {
+          setShowBorder(false)
+        }, 6000)
+      }
+
+      return () => {
+        navigation.setParams({ prevScreen: 'DASHBOARD' })
+      }
+    }, [navigation, post.id])
+  )
```

Comment:
I wonder if a `posted` param isn't enough to display a border? Do we need to validate if it's from notifications screen ?

Reply:
In fact it works this way. I'll remove it.